lib/cli.js

```diff
@@ -173,6 +175,10 @@ const meowOptions = {
         If the directory for the cache does not exist, make sure you add a trailing "/"
         on *nix systems or "\\" on Windows. Otherwise the path will be assumed to be a file.
 
+      --cache-strategy              [default: 'metadata']
+
+	    // TODO
```

Comment:
```suggestion
	    Strategy for the cache to use for detecting changed files. Can be either
	    "metadata" or "content".

        The "content" strategy can be useful in cases where the modification time of
        your files change even if their contents have not. For example, this can happen
        during git operations like "git clone" because git does not track file modification
        time.  
```

Let's adopt [ESLint's wording](https://eslint.org/docs/latest/user-guide/command-line-interface#--cache-strategy).

Let's also add it to these docs:
- https://github.com/stylelint/stylelint/blob/main/docs/user-guide/usage/cli.md
- https://github.com/stylelint/stylelint/blob/main/docs/user-guide/usage/options.md

Reply:
Thanks a lot!